plugins/rbac/src/components/CreateRole/PermissionPoliciesForm.tsx

```diff
@@ -126,6 +126,7 @@ export const PermissionPoliciesForm = ({
 
   const onAddConditions = (index: number, conditions?: ConditionsData) => {
     setFieldValue(`permissionPoliciesRows[${index}].conditions`, conditions);
+    setFieldValue(`permissionPoliciesRows[${index}].isResourced`, true);
```

Comment:
@debsmita1 ideally `isResourced` should be set while we are generating data for edit form similar to what is done for create form because we get `isResourced` as part of `RoleBasedPolicy` api. You can set this here https://github.com/janus-idp/backstage-plugins/blob/main/plugins/rbac/src/utils/rbac-utils.ts#L146.